.zuul.yaml

```diff
@@ -216,5 +229,21 @@
     name: opendev-watcher-edpm-pipeline
     openstack-check:
       jobs:
-        - openstack-meta-content-provider-master
-        - watcher-operator-validation-master
+        - openstack-meta-content-provider-master:
+            # Note(chandankumar): Add override checkout at project template
+            # pipeline so that RDO zuul discovers the job and complete the job
+            # graph in order to run the job.
+            override-checkout: main
+            required-projects:
+              - github.com/openstack-k8s-operators/watcher-operator
+            # Build watcher-operator always in meta content provider
+            vars:
+              cifmw_operator_build_operators:
+                - name: watcher-operator
+                  src: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/watcher-operator"
+                - name: openstack-operator
+                  src: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/openstack-operator"
+        - watcher-operator-validation-master:
+            override-checkout: main
+            required-projects:
+              - github.com/openstack-k8s-operators/watcher-operator
```

Comment:
let move forward with this patch as it is but

we should create a follow up to add branch matches so that this does not run on 
stable/2025.1 or other stabel brnahcess like it is here

https://review.opendev.org/c/openstack/watcher/+/946153

Reply:
to be clear we can create a version fo this for exposy if we want too but running the master veriosn with epxoy is not quite correct.